tests/brevitas/nn/test_recurrent.py

```diff
@@ -87,6 +88,11 @@ def test_rnn_quant_disabled_fwd(
         assert torch.isclose(out[0], ref_out[0], atol=ATOL).all()
         assert torch.isclose(out[1], ref_out[1], atol=ATOL).all()
 
+        # Test that the brevitas model can be saved/loaded without warning
```

Comment:
Could you please move this assert to a new test called test_quantrnn_state_dict where you instantiate a single QuantRNN with constant inp_size/hidden_size/etc. where you check only this assert?

Reply:
yes, done!